spec/scenic/statements_spec.rb

```diff
@@ -65,6 +84,18 @@ module Scenic
       end
     end
 
+    describe "update_view :materialized" do
+      it "raises an error because this is not supported" do
+        definition = instance_double("Definition").as_null_object
+        allow(Definition).to receive(:new).and_return(definition)
+
+        expect { connection.update_view(:name, version: 3, materialized: true) }.
```

Comment:
Line is too long. [81/80]<br>Place the . on the next line, together with the method name.